mavros/src/plugins/command.cpp

```diff
@@ -115,11 +116,22 @@ class CommandPlugin : public plugin::Plugin
       std::bind(
         &CommandPlugin::takeoff_cb, this, _1, _2,
         _3), rmw_qos_profile_services_default, srv_cg);
+    takeoff_local_srv =
+      node->create_service<mavros_msgs::srv::CommandLocalTOL>(
```

Comment:
I'm not entirely sure, but probably CommandTOLLocal will be better name, since everywhere 'local' used as a suffix.

Reply:
changed new service type name to `CommandTOLLocal` as requested.
marking as resolved.